### PR TITLE
fix #344:assembler-riscv64.cc:2457:15: warning: variable 'part' set but not used [-Wunused-but-set-variable] 

### LINKED ISSUE #344: assembler-riscv64.cc:2457:15: warning: variable 'part' set but not used [-Wunused-but-set-variable]
Seems an easy issue but should be fixed before upstreaming.

```
Done. Made 219 targets from 95 files in 139ms
ninja: Entering directory `out/riscv64.native.debug'                                                                                                                                                        [3041/4701] CXX obj/v8_base_without_compiler/assembler-riscv64.o                                                                                                                                            ../../src/codegen/riscv64/assembler-riscv64.cc: In static member function 'static int v8::internal::Assembler::li_estimate(int64_t, bool)':
../../src/codegen/riscv64/assembler-riscv64.cc:2457:15: warning: variable 'part' set but not used [-Wunused-but-set-variable]
 2457 |       int32_t part;                                                                                                                                                                                       |               ^~~~
```

### PR DESCRIPTION
**Pull request by Nekhlyudov**
fix #344

### CODE REVIEW

File: src/codegen/riscv64/assembler-riscv64.cc [hunk 1]
```diff
@@ -2454,16 +2454,16 @@ int Assembler::li_estimate(int64_t imm, bool is_get_temp_reg) {
         continue;
       }
       // The first 1 seen
-      int32_t part;
+      //int32_t part;
```
Comment (lazyparser) [inline src/codegen/riscv64/assembler-riscv64.cc:2457]: the style is bad. just delete the unused code, never comment out them. (the only case the code should live in comment is you have strong reason and write down as comments to explain)

**Comment by lazyparser** [comment]
Hi @Nekhlyudov 

nice try! 🎉

congrats on your first PR!

**Comment by luyahan** [comment]
Clean up your commit msg.Follow this:https://chris.beams.io/posts/git-commit/

File: src/codegen/riscv64/assembler-riscv64.cc (hunk 1 above)
Comment (Nekhlyudov) [reply src/codegen/riscv64/assembler-riscv64.cc:2457]: Ok, I learned that.

**Comment by Nekhlyudov** [comment]
I modified the code

**Comment by lazyparser** [comment]
> I modified the code

nice try.

try to rebase it again.

currently 5000+ files changed. looks wrong.

**Comment by lazyparser** [comment]
and I've noticed you just closed the PR.

no need to open an new PR. you can `push --force` to the branch.

**Pull request closed without merging**